Xamarin.Firebase.iOS.Crashlytics issue when uploading symbols on macOS 10.14 Mojave

A project that used to build fine before now fails after upgrading macOS to Mojave:
`[app-path]/packages/Xamarin.Firebase.iOS.Crashlytics.3.10.3.1/build/Xamarin.Firebase.iOS.Crashlytics.targets(53,3): error MSB3073: The command "/Users/thibaultdeckers/Library/Caches/XamarinBuildDownload/Fbrc-1.7.8/upload-symbols -gsp [app-path]/iOS/GoogleService-Info.plist -p ios bin/iPhone/Release/MyApp.iOS.app.dSYM" exited with code 8.`

When running this script as a standalone command:
`error: could not complete submission of dSYM at bin/iPhone/Release/MyApp.iOS.app.dSYM: Error Domain=com.crashlytics.mac.error-domain.process-dsym Code=4 "This version of OSX is not able to perform the necessary dSYM transformations." UserInfo={NSLocalizedFailureReason=This version of OSX is not able to perform the necessary dSYM transformations.}`

It seems Fabric added support for Mojave with version 1.7.11, but Xamarin.Firebase.iOS.Crashlytics v3.10.3.1 still relies on Fabric v1.7.8. An upgrade would be much appreciated.

--- a/Firebase.Crashlytics/component/component.yaml
+++ b/Firebase.Crashlytics/component/component.yaml
@@ -1,4 +1,4 @@
-version: 3.10.3.1
+version: 3.10.9.0
 name: Firebase Crashlytics for iOS
 id: firebaseioscrashlytics
 publisher: Xamarin Inc.
@@ -15,7 +15,7 @@ libraries:
 is_shell: true
 packages:
   ios-unified:
-  - Xamarin.Firebase.iOS.Crashlytics, Version=3.10.3.1
+  - Xamarin.Firebase.iOS.Crashlytics, Version=3.10.9.0
 samples:
 - name: Crashlytics Sample
   path: ../samples/CrashlyticsSample/CrashlyticsSample.sln

--- a/Firebase.Crashlytics/externals/Podfile
+++ b/Firebase.Crashlytics/externals/Podfile
@@ -1,15 +1,15 @@
 =begin
 Last run Crashlytics installed:
-* Crashlytics (3.10.3)
-* Fabric (1.7.8)
+* Crashlytics (3.10.9)
+* Fabric (1.7.13)
 
 Check if main version or subversion number has changed. 
 If yes, please, update *.targets files located in binding 
 projects, also, update Podfile files if needed.
 
 In in Firebase.Crashlytics binding project, you can find the .targets file of:
-* Crashlytics (3.10.3)
-* Fabric (1.7.8)
+* Crashlytics (3.10.9)
+* Fabric (1.7.13)
 
 =end
 
@@ -19,6 +19,6 @@ platform :ios, '8.0'
 install! 'cocoapods', :integrate_targets => false
 
 target 'Crashlytics' do
-	pod 'Crashlytics', '3.10.3'
-	pod 'Fabric', '1.7.8'
+	pod 'Crashlytics', '3.10.9'
+	pod 'Fabric', '1.7.13'
 end

--- a/Firebase.Crashlytics/nuget/Xamarin.Firebase.iOS.Crashlytics.nuspec
+++ b/Firebase.Crashlytics/nuget/Xamarin.Firebase.iOS.Crashlytics.nuspec
@@ -3,7 +3,7 @@
   <metadata>
     <id>Xamarin.Firebase.iOS.Crashlytics</id>
     <title>Firebase APIs Crashlytics iOS Library</title>
-    <version>3.10.3.1</version>
+    <version>3.10.9.0</version>
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>

--- a/Firebase.Crashlytics/source/Firebase.Crashlytics/Firebase.Crashlytics.targets
+++ b/Firebase.Crashlytics/source/Firebase.Crashlytics/Firebase.Crashlytics.targets
@@ -2,9 +2,9 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
 		<_FirebaseCrashlyticsAssemblyName>Firebase.Crashlytics, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null</_FirebaseCrashlyticsAssemblyName>
-		<_FirebaseCrashlyticsItemsFolder>FCrshlytcs-3.10.3</_FirebaseCrashlyticsItemsFolder>
+		<_FirebaseCrashlyticsItemsFolder>FCrshlytcs-3.10.9</_FirebaseCrashlyticsItemsFolder>
 		<_FirebaseCrashlyticsSDKBaseFolder>$(XamarinBuildDownloadDir)$(_FirebaseCrashlyticsItemsFolder)\iOS\</_FirebaseCrashlyticsSDKBaseFolder>
-		<_FabricItemsFolder>Fbrc-1.7.8</_FabricItemsFolder>
+		<_FabricItemsFolder>Fbrc-1.7.13</_FabricItemsFolder>
 		<_FabricSDKBaseFolder>$(XamarinBuildDownloadDir)$(_FabricItemsFolder)\iOS\</_FabricSDKBaseFolder>
 		<_FabricScriptName>upload-symbols</_FabricScriptName>
 
@@ -20,11 +20,11 @@
 	</PropertyGroup>
 	<ItemGroup Condition="('$(OutputType)'!='Library' OR '$(IsAppExtension)'=='True')">
 		<XamarinBuildDownload Include="$(_FirebaseCrashlyticsItemsFolder)">
-			<Url>https://kit-downloads.fabric.io/cocoapods/crashlytics/3.10.3/crashlytics.zip</Url>
+			<Url>https://kit-downloads.fabric.io/cocoapods/crashlytics/3.10.9/crashlytics.zip</Url>
 			<Kind>Zip</Kind>
 		</XamarinBuildDownload>
 		<XamarinBuildDownload Include="$(_FabricItemsFolder)">
-			<Url>https://kit-downloads.fabric.io/cocoapods/fabric/1.7.8/fabric.zip</Url>
+			<Url>https://kit-downloads.fabric.io/cocoapods/fabric/1.7.13/fabric.zip</Url>
 			<Kind>Zip</Kind>
 		</XamarinBuildDownload>
 		<NativeReference Include="$(_FirebaseCrashlyticsSDKBaseFolder)Crashlytics.framework">

--- a/Firebase.Crashlytics/source/Firebase.Crashlytics/Properties/AssemblyInfo.cs
+++ b/Firebase.Crashlytics/source/Firebase.Crashlytics/Properties/AssemblyInfo.cs
@@ -26,7 +26,7 @@ using Foundation;
 // and "{Major}.{Minor}.{Build}.*" will update just the revision.
 
 [assembly: AssemblyVersion ("1.0.0.0")]
-[assembly: AssemblyFileVersion ("3.10.3.0")]
+[assembly: AssemblyFileVersion ("3.10.9.0")]
 
 // The following attributes are used to specify the signing key for the assembly, 
 // if desired. See the Mono documentation for more information about signing.

